demo/demo3.html

```diff
@@ -52,11 +53,11 @@
 
   <app-drawer-layout fullbleed>
 
-    <div drawer>
+    <app-drawer>
       <app-toolbar>App name</app-toolbar>
-    </div>
+    </app-drawer>
 
-    <app-header-layout main has-scrolling-region>
+    <app-header-layout has-scrolling-region>
 
       <app-header condenses reveals effects="waterfall">
         <app-toolbar>
```

Comment:
When I tried to run the demo, the main content shows empty.  Can you check to see if is related to the changes?